client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/patterns-data.ts

```diff
@@ -188,9 +194,52 @@ const useSectionPatterns = () => {
 	const portfolio = translate( 'Portfolio' );
 	const posts = translate( 'Posts' );
 
+	const postsPatterns = [
+		{
+			id: 5645,
+			name: 'Four Recent Blog Posts',
+			category: posts,
+		},
+		{
+			id: 1784,
+			name: 'Recent Posts',
+			category: posts,
+		},
+		{
+			id: 8421,
+			name: 'Grid of posts 2x3',
+			category: posts,
+		},
+		{
+			id: 8435,
+			name: 'Grid of Posts 3x2',
+			category: posts,
+		},
+		{
+			id: 7996,
+			name: 'Grid of Posts 4x2',
+			category: posts,
+		},
+		{
+			id: 8437,
+			name: 'List of posts',
+			category: posts,
+		},
+		{
+			id: 3213,
+			name: 'Latest podcast episodes',
+			category: posts,
+		},
+	];
+
+	const addPostsPatternsInWriteFlow = () => ( intent === SiteIntent.Write ? postsPatterns : [] );
```

Comment:
What do you mean by sort by categories?

Reply:
Like this

```js
if ( intent === SiteIntent.Write ) {
  patterns.sort( ( a, b ) => {
    if ( a.category === b.category ) {
      return 0;
    } else if ( a.category === posts ) {
      return -1;
    } else if ( b.category === posts ) {
      return 1;
    }

    return 0;
  } );
}
```

Reply:
I see. Thanks. I thought about that. I think that is less flexible because we might want to control the position of those patterns on non-write flows.

Reply:
I think the position of those patterns on non-write flows is always the same, and we prioritize them only on write flows.

Reply:
Let's order all the patterns by categories alphabetically, as in the editor's pattern inserter, and move the Posts to the first position for the Write flow. I've pushed the changes.